### ISSUE TITLE
Document community recommended community moderation policies

### ISSUE THREAD

**Issue by silky** [haskellfoundation/haskellfoundation.github.io#345]
Following a discussion that has come up; it could be very useful if we were to have a "recommended" set of moderation policies that moderators in various communities could follow.

At a first pass, we can ask that the [Guidelines For Respectful Communication](https://haskell.foundation/guidelines-for-respectful-communication/) are used, and then we can recommend some moderation practices that we agree on.

To open the discussion; perhaps we could start with:

1. Moderators will actively enforce the guidelines,
2. We recommend reaching out to the poster and asking them to re-word/reconsider in line with the guidelines; in terms of acknowledging the other persons points; kindly expanding on disagreements, attempting to be curious about disagreements.
3. On occasion of repeated breaches, posts should be removed or hidden,
4. Flagging by community members should be encouraged,
5. There should be a regular process to raise community members into a pool of moderators; potentially also rotating moderator duties every year or two, to reduce burn-out,
6. Reminding users that moderation and respectful behaviour is a community effort,
7. Banning users should only be considered after repeated violations of the guidelines and an ability to reach an agreement in private conversation; all attempts should be made to avoid banning.

The wording of the above is not perfect; and probably missing many things; if you have suggestions please note them, and then we can attempt to draft something more complete!

**Comment by goldfirere** [comment]
Yes. I think these kinds of guidelines for moderation would be useful, and the HF is well placed to host such a document. However, such a thing will be useful only if it has the support of the several online communities we have (or at least some subset of them). So while drafting the document is important work, the harder piece (to me) is getting wide agreement on it and then some level of adoption.

In truth, I think this is best done after we have a new ED who can plow forward with that sometimes-delicate work. Maybe it makes sense to draft the document in this interim period, but I would be hesitant to put it on the website, say, without already working out buy-in from key constituencies.

Thanks for prodding this conversation!

**Comment by silky** [comment]
Sounds reasonable; I've added some Todo items around seeking feedback from some existing communities; if anyone knows others that would be interested, please add them!
